src/components/Map/MarkerCluster.tsx

```diff
@@ -0,0 +1,47 @@
+import L from "leaflet";
+import "leaflet.markercluster/dist/leaflet.markercluster";
+import "leaflet.markercluster/dist/MarkerCluster.css";
+import "leaflet.markercluster/dist/MarkerCluster.Default.css";
+import PropTypes from "prop-types";
+import { useEffect } from "react";
+import { useLeaflet } from "react-leaflet";
+import "./MarkerCluster.css";
+
+const mcg = L.markerClusterGroup();
+
+const customMarker = new L.Icon({
+  iconUrl: require("~/assets/icons/FarmMarker@2x.png"),
+  iconSize: [50, 60],
+  iconAnchor: [10, 41],
+  popupAnchor: [2, -40]
+});
+
+const MarkerCluster = ({ geoJson }) => {
```

Comment:
Hier dann die Props als TypeScript-Type hinzufügen.

```
type Props = {
  geoJson: Record<string, any>
}
```
`Record<string, any>` entspricht einem Objekt mit **string** als Keys und **any** als Values. Sprich: jedes Objekt ist valide, sofern dessen Keys jeweils vom Type **string** und deren Values jeweils vom Type **any** sind.

und in Z19 dann:
`const MarkerCluster = ({ geoJson }: Props) => {`